pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -363,12 +363,69 @@ func TestPriorityQueue_MoveAllToActiveOrBackoffQueue(t *testing.T) {
 	q.Add(&medPriorityPod)
 	q.AddUnschedulableIfNotPresent(q.newQueuedPodInfo(&unschedulablePod), q.SchedulingCycle())
 	q.AddUnschedulableIfNotPresent(q.newQueuedPodInfo(&highPriorityPod), q.SchedulingCycle())
+	// sleep 1 second so that pods is guaranteed to be added into unschedulableQ first and then
+	// the move request is happened later, otherwise there is potential a test flake here, as the
+	// move request has a chance to be finished before the pods get into the unschedulableQ.
+	// In that case, the pods will be moved to backoffQ even `QueuedPodInfo.Attempts` is zero.
+	time.Sleep(1000 * time.Millisecond)
```

Comment:
Do not do this. Instead, create the queue with a fake clock: `q := NewPriorityQueue(newDefaultQueueSort(), WithClock(clock.NewFakeClock(time.Now())))`

Reply:
Done